docs/kubernetes.core.k8s_info_module.rst

```diff
@@ -702,6 +702,19 @@ Examples
         wait_sleep: 10
         wait_timeout: 360
 
+    - name: Wait for OpenShift bootstrap to complete
+      kubernetes.core.k8s_info:
+        kind: ConfigMap
+        name: bootstrap
+        namespace: kube-system
+      register: ocp_bootstrap_status
+      until: ocp_bootstrap_status.resources is defined and
+        (ocp_bootstrap_status.resources | length > 0) and
+        (ocp_bootstrap_status.resources[0].data.status is defined) and
+        (ocp_bootstrap_status.resources[0].data.status == 'complete')
```

Comment:
```suggestion
    api_version: v1
    kind: ConfigMap
    name: bootstrap
    namespace: kube-system
    register: ocp_bootstrap_status
    until: >
      ocp_bootstrap_status.resources is defined and
      (ocp_bootstrap_status.resources | length > 0) and
      (ocp_bootstrap_status.resources[0].data.status is defined) and
      (ocp_bootstrap_status.resources[0].data.status == 'complete')
```
These changes are possibly a bit nitpicky but here's why I suggested them: 
- Even though `api_version` defaults to `v1`, it might be good to document that here to be explicit. 
- Adding the `>` increases readability.

Reply:
Thanks. 👍🏻